test/src/example.cpp

```diff
@@ -94,6 +103,10 @@ Viewport *Example::return_something_const() const {
 	return nullptr;
 }
 
+ExampleRef *Example::return_extended_ref() const {
```

Comment:
Shouldn't this return a `Ref<ExampleRef>`? I think this is more commonly used since it actually keeps track of the reference count.

Reply:
Currently `Ref<Object>` bindings are not working. Probably just missing the usual encode/convert functions. I'd rather fix that in another PR (this form should work in any case).

Reply:
Ref only works with classes extended from `RefCounted`, nodes aren't ref counted so you can't use Ref with the Example class.

Reply:
owh never mind, I missed we had a wrapper ExampleRef class... Did you try in combination with my PR that fixes the copy constructor?